Wiki/FAQ.md

```diff
@@ -0,0 +1,652 @@
+syncing to the pool. Then you can add new projects from the page of the
+pool.
+
+## I have connections but the number of blocks does not increase. What can I do?
+
+If you have connections you should be fine and the wallet will sort
+itself out. If after a few hours your number of blocks still doesn't increase check to make sure you're using the latest version of the wallet.
+
+## I had a magnitude for some time but it dropped to 0. What happened?
+
+The reason for this is most likely that you have to resend your beacon,
+which should be done every 6 months. For this either unlock your wallet
+so that it sends automatically or type `advertisebeacon` in the
+console.
+
+# Gridcoin & Science
+
+## Why is Gridcoin (GRC) special?
+
+Gridcoin allows  Nearly all computing power (up to
+98% efficiency) can be directed at beneficial scientific and medical
+research while crunching Gridcoin.
+
+See list of [Advantages &
+Features](Advantages_&_Features "wikilink").
+
+## How could 'useful' work for a cryptocurrency be verified without repeating the work or trusting a central server?
+
+In Gridcoin, work units are given out by distributed project nodes
+within the BOINC network which use their independent calculation to
+reward research. Gridcoin compares participants in each project by their
+subsidiary credits earned in BOINC to measure performance in relation to
+dynamic inner project network average.
+
+BOINC has been rewarding scientific work units in a credit system since
+2003. In case a project maintainer maliciously decides to give out more
+credits than appropriate, it only affects the inner project competition.
+The maximum share of new Gridcoins for this project stays the same
+percentage (subject to number of total projects). And projects can be
+removed by a blockchain vote.
+
+Security in Gridcoin is derived from Blackcoin's industry-leading
+Proof-of-Stake.
+
+# Crunching (Research & Mining Equivalent)
+
+## How do I start crunching?
+
+First, [Get some coins](Get_some_coins "wikilink"), then see [Mining
+setup](Mining-Setup "wikilink").
+
+## My CPID is not in the network, What do I do?
+
+If you are in the pool, this is normal. If you are attempting to
+solo-stake, go through the following:
+
+If you have just begun your BOINC journey it is quite possible your work
+units haven't been verified by the necessary servers. Don't Panic,
+Gridcoin is a game of patience. Just relax, take a day or two, and see
+if the problem persists. If so, proceed to the following steps.
+
+To ensure your wallet is fully unlocked, lock your wallet with a
+passphrase (make it LONG, a password manager like KeePass is
+recommended) and then unlock it with "for staking only" UNticked.
+
+Next, go to Help -\> Debug Window, go to the Console tab and type
+`advertisebeacon` without the quotes.
+
+This will (if successful) send a small transaction at the cost of
+-0.00011 GRC. This should add your CPID with the next superblock.

```

Comment:
this number will change in Fern

Reply:
So what should I change it to or should I just remove the number?

Reply:
I would change it to 1 GRC fee.  @cyrossignol can correct me if I'm wrong on that number